packages/gatsby-link/src/index.js

```diff
@@ -227,3 +227,7 @@ export default React.forwardRef((props, ref) => (
 export const navigate = (to, options) => {
   window.___navigate(rewriteLinkPath(to, window.location.pathname), options)
 }
+
+// This is to deal with microbundle not adding this property were converting to commonjs
+// while babel interops depend on existence of this export
+export const __esModule = true
```

Comment:
This as-is cause problems elsewhere (see failing unit tests). I can probably fix it going more like https://github.com/developit/microbundle/issues/306#issuecomment-459103476 but will pause on it, until I get general feedback on "manually" adding `__esModule` export